heading_capitalization.js

```diff
@@ -0,0 +1,28 @@
+module.exports = [{
```

Comment:
From where the file gets consumed? Is it picked up by naming convention? Could we maybe prefix it with something like ".markdown*" as well?
Could you at least put a comment in the file for what it is used